[2024-10-21T23:38:49.986+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-10-21T23:38:49.997+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: datapipeline_new.anamolies_vals_train manual__2024-10-21T23:38:23.170304+00:00 [queued]>
[2024-10-21T23:38:50.001+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: datapipeline_new.anamolies_vals_train manual__2024-10-21T23:38:23.170304+00:00 [queued]>
[2024-10-21T23:38:50.001+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-10-21T23:38:50.006+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): anamolies_vals_train> on 2024-10-21 23:38:23.170304+00:00
[2024-10-21T23:38:50.023+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=147) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-21T23:38:50.028+0000] {standard_task_runner.py:63} INFO - Started process 156 to run task
[2024-10-21T23:38:50.029+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'datapipeline_new', 'anamolies_vals_train', 'manual__2024-10-21T23:38:23.170304+00:00', '--job-id', '26', '--raw', '--subdir', 'DAGS_FOLDER/dag_script.py', '--cfg-path', '/tmp/tmp2i86k2ae']
[2024-10-21T23:38:50.032+0000] {standard_task_runner.py:91} INFO - Job 26: Subtask anamolies_vals_train
[2024-10-21T23:38:50.065+0000] {task_command.py:426} INFO - Running <TaskInstance: datapipeline_new.anamolies_vals_train manual__2024-10-21T23:38:23.170304+00:00 [running]> on host 6fd357fee1dd
[2024-10-21T23:38:50.266+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Srilakshmi_P' AIRFLOW_CTX_DAG_ID='datapipeline_new' AIRFLOW_CTX_TASK_ID='anamolies_vals_train' AIRFLOW_CTX_EXECUTION_DATE='2024-10-21T23:38:23.170304+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-21T23:38:23.170304+00:00'
[2024-10-21T23:38:50.267+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-10-21T23:38:50.274+0000] {logging_mixin.py:188} INFO - Data loaded from /opt/***/dags/DataPreprocessing/src/data_store_pkl_files/train_data/no_null_train_data.pkl.
[2024-10-21T23:38:50.277+0000] {logging_mixin.py:188} INFO - Outliers in 'pm25' replaced with median value 5.7.
[2024-10-21T23:38:50.277+0000] {logging_mixin.py:188} INFO - Negative values in 'pm25' replaced with 0.
[2024-10-21T23:38:50.278+0000] {logging_mixin.py:188} INFO - Cleaned DataFrame saved as '/opt/***/dags/DataPreprocessing/src/data_store_pkl_files/train_data/no_anamoly_train_data.pkl'.
[2024-10-21T23:38:50.279+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-10-21T23:38:50.279+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-10-21T23:38:50.282+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=datapipeline_new, task_id=anamolies_vals_train, run_id=manual__2024-10-21T23:38:23.170304+00:00, execution_date=20241021T233823, start_date=20241021T233849, end_date=20241021T233850
[2024-10-21T23:38:50.302+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-10-21T23:38:50.396+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-10-21T23:38:50.398+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
